src/routes/apiDirectory.js

```diff
@@ -80,6 +81,7 @@ router.use('/national-center', nationalCenterRouter);
 router.use('/communication-logs', communicationLogRouter);
 router.use('/monitoring', monitoringRouter);
 router.use('/courses', coursesRouter);
+router.use('/ssdi', ssdiRouter);
```

Comment:
Do we want to expose these routes yet? Also, should it be namespaced under admin?

Reply:
I would like to make the end point available, so it could be used sooner then later. I could see it moved under admin for now.

Reply:
I think moving it into admin moves the bar to getting it out there much lower